tools/edgeDeviceFiles.sh

```diff
@@ -48,7 +49,9 @@ Required Environment Variables:
 Required Environment Variables if specify -r:
```

Comment:
Typo "...Variables if -r is specified:" Again there are several places to change